scripts/cve-parsing/finalPublishedDates

```diff
@@ -0,0 +1,89 @@
+CVE-2002-0392.yml,2002-07-03T04:00Z
```

Comment:
This file shouldn't have been edited.

Reply:
I think it was my editor added stuff. It was un intentional